src/Auth/EnvMomentoTokenProvider.php

```diff
@@ -19,7 +19,7 @@ public function __construct(
         ?string $trustedCacheEndpointCertificateName = null
     )
     {
-        $authToken = getenv($envVariableName);
+        $authToken = $_SERVER[$envVariableName];
```

Comment:
This will crash if the key does not exist in the server super global. One should add `?? false`.

```
$authToken = $_SERVER[$envVariableName] ?? false;
```